README.md

```diff
@@ -58,6 +58,18 @@ class Query(graphene.ObjectType):
 schema = graphene.Schema(query=Query)
 ```
 
+We need a database session first:
```

Comment:
I would move this up to the "Here is a simple SQLAlchemy model:" part and add 
```python
# We will need this for querying
Base.query = db_session.query_property()
```
from the docs, since that is essential to using graphene-sqa.